scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -977,13 +978,18 @@ void pingTaskNode(RunningTaskData taskData) {
         } catch (Throwable t) {
             RunningTaskData runningTask = jobs.getRunningTask(task.getId());
             if (runningTask != null) {
-                int attempts = runningTask.increaseAndGetPingAttempts();
-                if (attempts > PASchedulerProperties.SCHEDULER_NODE_PING_ATTEMPTS.getValueAsInt()) {
-                    tlogger.info(task.getId(), "node failed", t);
-                    restartTaskOnNodeFailure(task);
+                if (NodesPinger.pingNode(runningTask.getNodes().get(0)) == -1) {
```

Comment:
Why reusing `NodesPinger#pingNode`? I mean, this last makes up to 10 remote calls if there is no connection issue (and this value is hardcoded :fire:). Is this really needed?